sky/backends/onprem_utils.py

```diff
@@ -492,6 +493,11 @@ def check_local_cloud_args(cloud: Optional[str] = None,
                 '`cloud: local` or no cloud in YAML or CLI args.')
         return True
     else:
+        if backend_utils.is_public_cloud_disabled():
+            check.check(quiet=True)
+            if backend_utils.is_public_cloud_disabled():
+                raise click.UsageError('Cloud access is not set up. '
```

Comment:
nit: how about using `RuntimeError` instead, because the `click.UsageError` will print some confusing strings for the usage of `sky launch`, but the user does not use the `sky launch` in the wrong way.
```
Usage: sky launch [OPTIONS] [ENTRYPOINT]...
Try 'sky launch -h' for help.
```

Reply:
Done, thanks!